jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -587,6 +587,13 @@ void java_bytecode_convert_methodt::convert(
   method_symbol.location=m.source_location;
   method_symbol.location.set_function(method_identifier);
 
+  code_typet::exceptionst &exceptions_list = code_type.throws_exceptions();
+  for(const auto &p : m.throws_exception_table)
```

Comment:
I dislike using just `p` a variable name. Having the type (if it is short) can also be handy. Maybe `const irep_idt &exception_name` here would be good?